var/spack/repos/builtin/packages/glm/package.py

```diff
@@ -12,8 +12,26 @@ class Glm(CMakePackage):
     """
 
     homepage = "https://github.com/g-truc/glm"
-    url = "https://github.com/g-truc/glm/archive/0.9.7.1.tar.gz"
+    url = "https://github.com/g-truc/glm/archive/0.9.9.8.tar.gz"
 
+    version('0.9.9.8', sha256='7d508ab72cb5d43227a3711420f06ff99b0a0cb63ee2f93631b162bfe1fe9592')
     version('0.9.7.1', sha256='285a0dc8f762b4e523c8710fbd97accaace0c61f45bc8be2bdb0deed07b0e6f3')
 
-    depends_on('cmake@2.6:', type='build')
+    depends_on('cmake@3.2:', type='build')
+
+    # CMake install target was removed in version 0.9.9.6
+    @when('@0.9.9.6:')
+    def cmake(self, spec, prefix):
+        pass
+
+    @when('@0.9.9.6:')
+    def build(self, spec, prefix):
+        pass
+
+    @when('@0.9.9.6:')
+    def install(self, spec, prefix):
+        mkdirp(prefix.include.glm)
+        ignore_cmakelists = lambda p: p.endswith('CMakeLists.txt')
+        install_tree('glm', prefix.include.glm, ignore=ignore_cmakelists)
+        mkdirp(prefix.lib64.cmake)
+        install_tree('cmake', prefix.lib64.cmake)
```

Comment:
This doesn't look like it depends on cmake at all, it looks like it just copies files over. Why was the cmake constraint added in this PR?

Reply:
Let me double-check that, it's indeed weird.

Reply:
Ok, this `cmake@3.2:` was introduced in version 0.9.9.0. I have repushed.

I don't think I can remove the CMake dependency completely as of version 0.9.9.6 (or at least not when inheriting from `CMakePackage`.